### DISCUSSION TITLE
How to generate $\succ^{h(i)}$ in LaTeX

### DISCUSSION THREAD

**Discussion by jlduu** [typst/typst#4518] [General]
I use \$ succ^h(i) \$ in Typst, it generates as $\overset{h(i)}{\succ}$, but I want $\succ^{h(i)}$ instead.

**Comment by jamesrswift** [comment]
Have you tried `script(succ)^h(i)`?

**Reply by jlduu** [reply]
Many thanks

**Comment by Mc-Zen** [comment]
Hi @jlduu ,

This can be achieved with 
```typ
$attach(succ, tr: h(i))$
```

![image](https://github.com/typst/typst/assets/52877387/c6259cfd-1ac5-4fee-8df3-5f07c98bced1)

The function `attach` allows you to specify `t` for top as well as `tr` for top right (and many more).

**Reply by jlduu** [reply]
Many thanks
